README.md

```diff
@@ -15,6 +15,7 @@ Supported Formats
 | FLAC | "flac" | YES | - | flac | [claxon](https://crates.io/crates/claxon) |
 | Ogg Vorbis | "ogg", "oga" | YES | - | ogg_vorbis | [lewton](https://crates.io/crates/lewton) |
 | WAV | "wav", "wave" | YES | - | wav | [hound](https://crates.io/crates/hound) |
+| ALAC (within CAF) | YES | - | caf_alac | [caf](https://crates.io/crates/caf) [alac](https://crates.io/crates/alac) |
```

Comment:
Looks like the table is missing the extensions column:

<img width="747" alt="screen shot 2017-02-08 at 10 38 53 pm" src="https://cloud.githubusercontent.com/assets/4587373/22735812/88618f7e-ee4f-11e6-82b1-e18fbab1b66d.png">